easybuild/easyblocks/o/openblas.py

```diff
@@ -13,7 +13,7 @@
 from easybuild.tools.systemtools import POWER, get_cpu_architecture, get_shared_lib_ext
 from easybuild.tools.build_log import EasyBuildError, print_warning
 from easybuild.tools.config import ERROR
-from easybuild.tools.run import run_cmd, check_log_for_errors
+from easybuild.tools.run import run_shell_cmd, check_log_for_errors
```

Comment:
In #4433 we're also deprecating `check_log_for_errors`, do we really still need it here?

Reply:
replaced in 9661d45